app/controllers/post_controller.rb

```diff
@@ -0,0 +1,19 @@
+class PostController < ApplicationController
```

Comment:
If you run into the error `Uninitialized constant PostsController`, it's because you specified `resources :posts` in your config/routes.rb, so Rails is looking for the PostsController but yours is named PostController.